### PR TITLE
Tutorial updated with Nicla Vision added, new links and screenshots

### PR DESCRIPTION
**Pull request by jcarolinares**
…Grammar small fixes and redundancy check

## What This PR Changes
- Tutorial updated with Nicla Vision added, new links and screenshots

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jcarolinares** [comment]
I can confirm the new library is already available on the Arduino IDE.

**Comment by jcarolinares** [comment]
Symbolic link won't work, fixing it

File: content/hardware/06.nicla/boards/nicla-vision/tutorials/speech-recognition-engine
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/tutorials/speech-recognition-engine:1]: @jacobhylen looks like I'm doind the symbolik link wrong?

Comment (jacobhylen) [reply content/hardware/06.nicla/boards/nicla-vision/tutorials/speech-recognition-engine:1]: Pushed an updated symlink :)

**Comment by jcarolinares** [comment]
thanks @jacobhylen  :)

**Comment by jcarolinares** [comment]
Preview with symbolic link working

Merging

**Pull request merged**